packages/amplication-client/src/Project/ArchitectureConsole/ModelOrganizer.tsx

```diff
@@ -284,6 +304,15 @@ export default function ModelOrganizer() {
             />
           </div>
           <div className={`${CLASS_NAME}__body`}>
+            <ConfirmationDialog
+              isOpen={confirmApplyChangesDialog}
+              onDismiss={onApplyChangesConfirmationClicked}
+              message={`Your architecture tweaks are ready to be applied, allowing Amplication to generate its code.
+              To reset or fetch updates not in the current state, use the buttons in the top toolbar.`}
+              confirmButton={{ label: "Got it" }}
+              onConfirm={onApplyChangesConfirmationClicked}
+            ></ConfirmationDialog>
+
```

Comment:
Use a new component <ModelOrganizerChangesExistConfirmation/> and move all the logic including the useState and messages to it to keep the parent component lean